CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Added new [`roblox_suspicious_udim2_new` lint](https://kampfkarren.github.io/selene/lints/roblox_suspicious_udim2_new.html), which will warn when you pass in too few number of arguments to `UDim2.new`.
 - `roblox_incorrect_roact_usage` now lints for illegal `Name` property
 - Added `ignore_pattern` config to `global_usage`, which will ignore any global variables with names that match the pattern
+- `roblox_incorrect_roact_usage` now supports react-lua

```

Comment:
1. Everywhere you refer to this as "react-lua", call it Roact17. It's the name I think everyone should probably start using
2. Be more specific about what this entails--so that `key`, `children`, and `ref` are no longer linted against, and that `React` is accepted as a variable name